apps/common-app/src/examples/RuntimeTests/RuntimeTestsExample.tsx

```diff
@@ -12,6 +12,9 @@ import './tests/animations/withTiming/transformMatrices.test';
 import './tests/animations/withDecay/basic.test';
 import './tests/animations/withSpring/variousConfig.test';
 
+import './tests/animations/withDelay/keepSnapshot.test';
+import './tests/animations/withDelay/addDelays.test';
```

Comment:
Are you going to use the syntax that you used for `withSequence`? (https://github.com/software-mansion/react-native-reanimated/blob/89fe6461036fd483abfcee2835e130819098bac4/apps/common-app/src/examples/RuntimeTests/tests/animations/index.tsx)

Reply:
Yes!